test/test_utils.py

```diff
@@ -0,0 +1,15 @@
+from utils import utils
+
+
+class TestUtils(object):
+    def test_str_to_boolean(self):
+        cases = [
+            ("True", True),
+            ("False", False)
+        ]
+        for s, gt in cases:
+            assert utils.str_to_boolean(s) == gt
+
+    def test_get_timestamp(self):
+        stamp = utils.get_timestamp()
+        assert stamp.endswith('Z')
```

Comment:
According to PEP8, here we need a new line at the end of the file.

Reply:
I think we didn't consider the config file of vscode, in next PR, we can cover it in .gitignore.

Reply:
I think it's ok with me. just add a new line and merge it.